### PR TITLE
W-14046899: Update Flex 1.5 Header Removal policy doc

### PR DESCRIPTION
**Pull request by chris-campbell9**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.5/modules/ROOT/pages/policies-included-header-removal.adoc [hunk 1]
```diff
@@ -105,4 +104,4 @@ Because every header that matches a Header Removal policy parameter is removed f
 For example, the following parameter configuration removes any header starting with the word `Access-Control`, such as the `Access-Control-Allow-Credentials` and `Access-Control-Allow-Origin` headers from a message. The configuration also removes the `Cache-Control` header:
 
 .Inbound Header
-image::remove-inbound-header.png[]
```
Comment (aneisen-mulesoft) [inline gateway/1.5/modules/ROOT/pages/policies-included-header-removal.adoc:108]: why this change?

Comment (chris-campbell9) [reply gateway/1.5/modules/ROOT/pages/policies-included-header-removal.adoc:108]: It's actually not a change. For some reason GitHub marked it as delete and then marked it as add in the next line.

**Review by aneisen-mulesoft — APPROVED** [review]

**Review by dmerlob — APPROVED** [review]
